templates/uipath-orchestrator.template.yaml

```diff
@@ -1192,13 +1197,14 @@ Resources:
               - Effect: Allow
                 Action:
                   - "events:PutEvents"
-                Resource: !Sub 'arn:aws:events:${AWS::Region}:402050645054:event-bus/default'
-
+                Resource: !Sub 'arn:${AWS::Partition}:events:eu-west-1:402050645054:event-bus/telemetry'
```

Comment:
What's the purpose of this hardcoded AWS Account number here? Does it belong to UiPath?

Reply:
Yes, the account is used for telemetry.